Resources/Prototypes/Recipes/Lathes/security.yml

```diff
@@ -422,3 +422,21 @@
   completetime: 5
   materials:
     Plastic: 950
+
+- type: latheRecipe
+  id: ClothingEyesHudSecurity
+  result: ClothingEyesHudSecurity
+  name: security hud
+  completetime: 2
+  materials:
+    Glass: 200
+    Steel: 300
```

Comment:
duplicate

Reply:
There is no duplicate, they are differentiated by name ClothingEyesHudSecurity (427) and ClothingEyesHudMedSec (436)

Reply:
or did I misunderstand something